package com.oztechan.ccc.common.core.network.error

internal class NetworkException(cause: Throwable) : Throwable(cause)
